src/transformers/agents/prompts.py

```diff
@@ -480,3 +773,15 @@ def download_prompt(prompt_or_repo_id, agent_name, mode="run"):
 ```
 {plan_update}
 ```"""
+
+SUPPORTED_PLAN_TYPES = ["default", "structured"]
+
+plan_type_to_prompt_first = {
```

Comment:
Good idea to have a dict for clarity @plaggy !
I think we should rename these to make them more explicit: something like `PROMPTS_FOR_INITIAL_PLAN` and `PROMPTS_FOR_PLAN_UPDATE` ?